src/sync/sync_user.cpp

```diff
@@ -214,11 +214,11 @@ void SyncUser::update_access_token(std::string token)
             case State::Removed:
                 return;
             case State::LoggedIn:
-                m_access_token = token;
+                m_access_token = std::move(token);
                 break;
             case State::LoggedOut: {
                 sessions_to_revive.reserve(m_waiting_sessions.size());
-                m_access_token = token;
+                m_access_token = std::move(token);
```

Comment:
same situation here with using a moved from variable below

Reply:
I would also consider moving the `std::move` to within the `RealmJWT` constructor, and call it explicitly here:
`m_access_token = RealmJWT(token)`

Reply:
`token` is now only used to construct a `RealmJWT`, anything that needs to access `token` now just used `m_access_token.token` directly

`std::move` has also moved into the constructor of RealmJWT